roles/pulp_common/tasks/repos.yml

```diff
@@ -126,14 +126,23 @@
     - ansible_facts.distribution == "CentOS"
     - ansible_facts.distribution_major_version|int >= 8
 
-- name: "Add {{ __pulp_pkg_repo_name }} RPM repository"
-  yum_repository:
-    name: "{{ __pulp_pkg_repo_name }}"
-    description: "{{ __pulp_pkg_repo_name }}"
-    baseurl: "{{ pulp_pkg_repo }}"
-    gpgcheck: "{{ pulp_pkg_repo_gpgcheck }}"
-    gpgkey: "{{ __pulp_pkg_repo_gpgkey }}"
-    enabled: true
+- name: "Configure {{ __pulp_pkg_repo_name }} RPM repository"
+  block:
+    - name: "Create module_hotfixes config file"
+      copy:
+        dest: /etc/yum/pulpcore.conf
+        content: |
+          module_hotfixes=1
+
+    - name: "Add {{ __pulp_pkg_repo_name }} RPM repository"
+      yum_repository:
+        name: "{{ __pulp_pkg_repo_name }}"
+        description: "{{ __pulp_pkg_repo_name }}"
+        baseurl: "{{ pulp_pkg_repo }}"
+        gpgcheck: "{{ pulp_pkg_repo_gpgcheck }}"
+        gpgkey: "{{ __pulp_pkg_repo_gpgkey }}"
+        enabled: true
+        include: /etc/yum/pulpcore.conf
```

Comment:
the `include` indirection is needed because `yum_repository` doesn't support `module_hotfixes` parameter :roll_eyes:

Reply:
@evgeni Nice! I didn't know about the `include` option. That is a great way to solve the problem.